src/contention_detectors/signal_analyzers/drop.hpp

```diff
@@ -131,6 +97,49 @@ class SignalDropAnalyzer : public SignalAnalyzer {
    */
   void shiftBasePoints();
 
+  //! int64_t
+  //! How far in the past we look.
+  void setCfgWindowSize(int64_t cfgWindowSize) {
+    SignalDropAnalyzer::cfgWindowSize = cfgWindowSize;
+  }
+
+  //! int64_t
+  //! Maximum number of checkpoints we will have in our assurance detector.
+  //! Checkpoints are the reference assurance_test(base) points which we refer
+  //! to in the past when detecting drop or not.
+  //! It needs to be 0 < < WINDOW_SIZE
```

Comment:
We should check this condition in setter... but it makes unfortunate time coupling with setWindowSize function - could we make something with that?